### PR TITLE
Remove Bootstrap 3 support from components

### PR DESCRIPTION
**Pull request by simonihmig**
Some other related PRs will need to follow, e.g. for the default blueprint.

### CODE REVIEW

**Comment by simonihmig** [comment]
Failing tests for canary and beta (which is now also 4.0!) seem unrelated. The `Ember` global has been removed now, which some dependencies were still using I believe... 😔

File: addon/components/bs-accordion/item/title.hbs [hunk 1]
```diff
@@ -17,24 +17,15 @@
   </h2>
 {{else}}
   <div
-    class="{{if (macroCondition (macroGetOwnConfig "isBS3")) "panel-heading"}} {{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
+    class="{{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
```
Comment (jelhan) [inline addon/components/bs-accordion/item/title.hbs:20]: `macroCondition` isn't needed here anymore. The code is only run for BS4

**Review by jelhan — APPROVED** [review]
Looks good to me. Thanks for working on this one.

Disclaimer: I haven't reviewed the changes to tests. Nor if it is comprehensive.

File: addon/components/bs-accordion/item/title.hbs (hunk 1 above)
Comment (simonihmig) [reply addon/components/bs-accordion/item/title.hbs]: Nice catch, fixed! :+1:

**Pull request merged**